src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicSpawnerTag.java

```diff
@@ -0,0 +1,341 @@
+package com.denizenscript.depenizen.bukkit.objects.mythicmobs;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.Adjustable;
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.tags.TagRunnable;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.depenizen.bukkit.bridges.MythicMobsBridge;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import io.lumine.xikage.mythicmobs.adapters.AbstractLocation;
+import io.lumine.xikage.mythicmobs.adapters.bukkit.BukkitAdapter;
+import io.lumine.xikage.mythicmobs.mobs.MythicMob;
+import io.lumine.xikage.mythicmobs.spawning.spawners.MythicSpawner;
+import org.bukkit.World;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+public class MythicSpawnerTag implements ObjectTag, Adjustable {
+
+    // <--[language]
+    // @name MythicSpawnerTag Objects
+    // @group Depenizen Object Types
+    // @plugin Depenizen, MythicMobs
+    // @description
+    // A MythicSpawnerTag represents a MythicMobs spanwer in the world.
+    //
+    // These use the object notation "mythicspawner@".
+    // The identity format for a MythicSpawner is its name.
+    // For example, 'mythicspawner@AngrySludgeSpawner1'.
+    //
+    // -->
+
+    public static MythicSpawnerTag valueOf(String name) {
+        return valueOf(name, null);
+    }
+
+    @Fetchable("mythicspawner")
+    public static MythicSpawnerTag valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+        try {
+            string = string.replace("mythicspawner@", "");
+            if (!MythicMobsBridge.isMythicSpawner(string)) {
+                return null;
+            }
+            return new MythicSpawnerTag(MythicMobsBridge.getMythicSpawner(string));
+        }
+        catch (Exception e) {
+            return null;
+        }
+    }
+
+    public static boolean matches(String string) {
+        return valueOf(string) != null;
+    }
+
+    public MythicSpawnerTag(MythicSpawner spawner) {
+        if (spawner != null) {
+            this.spawner = spawner;
+        }
+        else {
+            Debug.echoError("Mythic Spawner referenced is null!");
```

Comment:
This error shouldn't be here. It's an internal java-level error if you do `new MythicSpawnerTag(null)`, and thus any cases that *might* be that should be null checked prior to constructing